SubscribeIT/ngDesk
# 319 list time stamp issue in list layout

#### Reference to issue (Required)

Closes #319 

#### Checklist (Required)


#### Description (Required)

The timestamp values were displayed in different timezone in list layout and one to many dailog list.

#### Manual Test Preformed (Required)
List the manual tests that were preformed and the steps to preform them
**There must be multiple test cases provided, each test case should include the exact and detailed steps to replicate the case**
**I have included 1, test cases here but there will be many time when more cases are required**

#### Test Case 1
1. Create date, time, date/time fields in any module like tickets.
2. Create a one-to-many relationship field for that ticket module.
3. Add the relationship field in the layouts.
4. Open the map of the relationship field, check whether the date types field values are same as compared to the list layout of the ticket module.
5. Screenshot:
![Screenshot from 2021-12-28 13-46-10](https://user-images.githubusercontent.com/89620702/147545592-12456548-72a5-41c6-b0d4-a10193b8389b.png)
![Screenshot from 2021-12-28 13-46-20](https://user-images.githubusercontent.com/89620702/147545599-ca3022cb-5502-48c4-8214-d338ee22c72f.png)


#### List of General Components affected (Required)
ngdesk-UI

#### Types of changes (Required)
